blog/_posts/2019-06-27-commit-of-the-month-june.md

```diff
@@ -0,0 +1,38 @@
+---
+slug: june-tilt-commit-of-the-month
+date: 2019-07-03T15:26:34.743Z
+author: dmiller
+layout: blog
+title: "Tilt Commit of the Month"
+subtitle: "June 2019 Edition"
+images:
+  - featuredImage.jpg
+image_type: "contain"
+tags:
+  - docker
+  - kubernetes
+  - microservices
+  - tilt
+  - commit
+keywords:
+  - docker
+  - kubernetes
+  - microservices
+  - tilt
```

Comment:
Should we have a "Commit of the Month" (/ "CotM") tag so that users can filter by that?

Reply:
FWIW we don't seem to expose these tags to users?